recipe/meta.yaml

```diff
@@ -1,29 +1,61 @@
-# Switch the version of this recipe based on the VC_VERSION environment variable.
-{% set VERSION = os.environ.get('VC_VERSION', '9')|string %}
-{% set runtime_years = {'9': '2008', '10': '2010', '14': '2015'} %}
-{% set runtime_year = runtime_years[VERSION] %}
+# version numbering is explained at https://stackoverflow.com/questions/42730478/version-numbers-for-visual-studio-2017-boost-and-cmake
+#     This version number follows the "platform toolset"
+{% set vcver="14" %}
+{% set vsyear="2015" %}
+{% set fullver="14.0.25420" %}
 
 package:
-  name: vc
-  version: {{ VERSION }}
+  name: vs{{ vsyear }}
+  version: {{ fullver }}
 
 build:
-  number: 0
-  track_features:
-    - vc{{ VERSION }}
-  skip: true  # [not win]
+  number: 11
+  skip: True  [not win]
 
-requirements:
-  run:
-    - vs{{ runtime_year }}_runtime
+outputs:
+  - name: vs{{ vsyear }}_{{ cross_compiler_target_platform }}
+    script: install_activate.bat
+    track_features:
+      - vc{{vcver}}
+    run_exports:
+      strong:
+        # compatible within major version.  This is MSFT's incrementing of the UCRT version.  It was
+        #    14.0 with VS2015, and 14.1 with VS2017 - if the major version changes, we're assuming
+        #    that binary compatibility breaks.
+        - {{ pin_subpackage('vc') }}
+        - {{ pin_subpackage('vs' ~ vsyear ~ '_runtime', max_pin=None) }}
+    about:
+      summary: Activation and version verification of MSVC {{ vcver }} (VS {{ vsyear }}) compiler
+      license: BSD 3-clause
 
-# We don't need to test anything here - this is purely a metapackage.
-test: {}
+  - name: vs{{ vsyear }}_runtime
+    script: install_runtime.bat
+    build:
+      no_link:
+        - |
+          *.dll
+        - Library/bin/*.dll
 
-about:
-  home: https://github.com/conda/conda/wiki/VC-features
-  license: BSD 3-Clause
-  summary: A meta-package to track VC features.
+  - name: vc
+    version: {{ vcver }}
+    track_features:
+      - vc{{vcver}}
+    requirements:
+      run:
+        # Set lower bound for runtime (no upper bound).  Upper bound is still present, but
+        #    it's part of the run_exports for the compiler: assume that when vcver's major version changes,
```

Comment:
> Upper bound is still present, but it's part of the run_exports for the compiler

I'm a little confused by this. the compiler recipe has only an upper bound on `vc`.

Reply:
The `vs2017` recipe from Anaconda does `{{ pin_subpackage('vs' + runtime_year + '_runtime') }}` whereas the `vs2015` does not.
https://github.com/AnacondaRecipes/aggregate/blob/cfda261b769dacfe38f89c71d91a077859014c47/vs2015/meta.yaml#L25
https://github.com/AnacondaRecipes/aggregate/blob/9635228cd0beb9f1113d3d3be29d278386fc7be4/vs2017/meta.yaml

Reply:
`run_exports` on `vs2015_runtime` was added to fix warnings from conda-build. (@msarahan added it only for vs2017. I'm adding it here for vs2015 as well.

Reply:
Okay, but what I mean is: https://github.com/conda-forge/vc-feedstock/pull/10/files#r349466866